src/backend/dx12/src/device.rs

```diff
@@ -73,13 +73,13 @@ fn shader_bytecode(shader: *mut d3dcommon::ID3DBlob) -> d3d12::D3D12_SHADER_BYTE
 }
 
 #[derive(Clone)]
-struct ViewInfo {
-    resource: *mut d3d12::ID3D12Resource,
-    kind: image::Kind,
-    flags: image::StorageFlags,
-    view_kind: image::ViewKind,
-    format: dxgiformat::DXGI_FORMAT,
-    range: image::SubresourceRange,
+pub(crate) struct ViewInfo {
+    pub(crate) resource: *mut d3d12::ID3D12Resource,
```

Comment:
given that this is used in the command buffer module now, I feel that we more and more collide with that other PR. Need to see which parts work good, and how to avoid sharing too much between those modules